config.py

```diff
@@ -1,37 +1,42 @@
 import numpy as np
 import h5py
+import yaml
 class Config(object):
```

Comment:
@Yufeng-shen 
Python3 uses the object class as default base class, so there is no need to specify it anymore.